device.mk

```diff
@@ -205,4 +205,26 @@ PRODUCT_PROPERTY_OVERRIDES += \
 #
 PRODUCT_DEFAULT_PROPERTY_OVERRIDES += \
     persist.sys.usb.config=mtp \
```

Comment:
We definitely do NOT want mtp enabled by default. We control that dynamically based on the settings app, and use the value of persist.sys.usb.config as what we set sys.usb.config in order to turn MTP off. I don't understand why changing this to mass_storage would cause any issues. Volumes aren't actually shared with the PC unless persist.sys.usb/config contains mass_storage AND the appropriate settings are enabled in the settings app.